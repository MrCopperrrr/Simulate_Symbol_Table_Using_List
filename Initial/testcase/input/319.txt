BEGIN
INSERT ap string
RPRINT
BEGIN
INSERT aq number
RPRINT
BEGIN
INSERT ar string
RPRINT
END
INSERT as number
RPRINT
END
INSERT at string
RPRINT
END
INSERT au number
RPRINT