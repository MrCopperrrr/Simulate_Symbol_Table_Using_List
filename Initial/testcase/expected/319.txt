success
ap//1
success
aq//2 ap//1
success
ar//3 aq//2 ap//1
success
as//2 aq//2 ap//1
success
at//1 ap//1
success
au//0